editor/src/themes/survey/SurveyHeader.scss

```
@@ -80,20 +81,38 @@
+      letter-spacing: -0.68px;
+      padding: 0;
+    }
+  }
+
   .welcome-title,
   .welcome-description {
-    font-size: $font-size-md;
-    letter-spacing: -0.017em;
-    text-align: left;
+    @include reg16;
     margin-bottom: 10px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)